README.md

```diff
@@ -1,24 +1,56 @@
 # Eden
 
-Eden is where [EVE](https://github.com/lf-edge/eve) and [Adam](https://github.com/lf-edge/adam) get tried and tested.
+Eden is the simpliest way to setup & test [EVE](https://github.com/lf-edge/eve) and [Adam](https://github.com/lf-edge/adam).
 
-Eden consists of a test harness and a series of integration tests implemented in Golang. Tests are structured as normal Golang tests by using ```_test.go``` nomenclature and be available for test runs using standard go test framework.
+Eden is inspired to become a Kubernetes for LFEdge 
 
-## Install Dependencies
+Eden is the easy way to deploy & communicate with Adam/Eve. You don't need any other project except Eden to try LFEdge. 
+
+Eden contains  series of integration tests implemented in Golang. Tests are structured as normal Golang tests by using ```_test.go``` nomenclature and be available for test runs using standard go test framework.
+
+## Install Prerequisites
 
 Install requirements from [eve](https://github.com/lf-edge/eve#install-dependencies)
 
 Also, you need to install `uuidgen`.
 
-## Running
+Your user needs to be able to run `docker` commands and able to access kvm.
```

Comment:
Let's just say: "You need to be able to run `docker` commands and able to access virtualization accelerators (KVM on Linux or machyve on Mac OS X)"